springboot/src/main/java/ca/jrvs/apps/trading/controller/QuoteController.java

```diff
@@ -0,0 +1,38 @@
+package ca.jrvs.apps.trading.controller;
+
+import ca.jrvs.apps.trading.model.domain.IexQuote;
+import ca.jrvs.apps.trading.service.QuoteService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@Controller
+@RequestMapping("/quote")
+public class QuoteController {
+
+  private QuoteService quoteService;
+
+  @Autowired
+  public QuoteController(QuoteService quoteService) {
+    this.quoteService = quoteService;
+  }
+
+  //@ApiOperation(value = "Show IexQuote", notes = "Show IexQuote for a given ticker/symbol")
+  //@ApiResponses(value = {@ApiResponse(code = 404, message = "ticker not found")})
```

Comment:
try to understand what are these commented annotations do. (tip they are springfox annotations rather than springboot)